python/CMakeLists.txt

```diff
@@ -289,7 +289,7 @@ else ()
       set_tests_properties(python-tox-test
         PROPERTIES
         PASS_REGULAR_EXPRESSION "Totals: .* ignored, 0 failed"
-        FAIL_REGULAR_EXPRESSION "ERROR:[ ]+py[0-9]*: commands failed")
```

Comment:
This is probably a good change irrespective of anything else! (it's just more future proof)